packages/react-events/src/Press.js

```diff
@@ -571,11 +570,56 @@ function removeRootEventTypes(
 ): void {
   if (state.addedRootEvents) {
     state.addedRootEvents = false;
-    state.allowPressReentry = false;
     context.removeRootEventTypes(rootEventTypes);
   }
 }
 
+function getTouchById(
+  nativeEvent: TouchEvent,
+  pointerId: null | number,
+): null | Touch {
+  const changedTouches = nativeEvent.changedTouches;
+  for (let i = 0; i < changedTouches.length; i++) {
+    const touch = changedTouches[i];
+    if (touch.identifier === pointerId) {
+      return touch;
+    }
+  }
+  return null;
+}
+
+function getTouchTarget(context: ReactResponderContext, touchEvent: Touch) {
+  const doc = context.getActiveDocument();
+  return doc.elementFromPoint(touchEvent.clientX, touchEvent.clientY);
+}
+
+function updatePressResponderRegion(
```

Comment:
nit: should probably be called updateIsPressWithinResponderRegion and it looks like the `isPressWithinResponderRegion` function is only used within this new function now, so maybe that can be inlined here?